web/src/invite_stream_picker_pill.ts

```diff
@@ -0,0 +1,63 @@
+import * as input_pill from "./input_pill";
+import {set_up_stream} from "./pill_typeahead";
+import * as stream_data from "./stream_data";
+import * as stream_pill from "./stream_pill";
+import type {CombinedPillItem} from "./typeahead_helper";
+
+type SetUpPillTypeaheadConfig = {
+    pill_widget: stream_pill.StreamPillWidget;
+    $pill_container: JQuery;
+};
+
+function create_item_from_stream_name(
+    stream_name: string,
+    current_items: CombinedPillItem[],
```

Comment:
I think this can still reuse the `stream_pill` function by passing a parameter like `stream_prefix_required` and `show_stream_sub_count`.

Reply:
Done.